definitions/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -12601,40 +12601,400 @@
   },
   {
     "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "etInitialConsiderationBeforeYouStart",
+    "CaseFieldID": "initialConsiderationBeforeYouStart",
```

Comment:
legalrepsolicitor shouldn't have access to IC event or fields

Reply:
even if it is D

Reply:
Removed